### PR TITLE
[Feature] Navbar

### PR DESCRIPTION
**Pull request by falbas**

### CODE REVIEW

File: src/components/modules/Navbar/NavItem.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  href: string
+  className?: string
+  target?: HTMLAttributeAnchorTarget
+}
+
+export const NavItem = ({ children, href, className, target }: Props) => {
+  return (
+    <li className={`${className} mt-8 lg:ml-10 lg:mt-0 xl:ml-16 xl:mt-0`}>
+      <Link href={href}>
+        <a target={target}>{children}</a>
```
Comment (gregoriusjimmy) [inline src/components/modules/Navbar/NavItem.tsx:15]: why there is an a tag inside Link?

Comment (gregoriusjimmy) [inline src/components/modules/Navbar/NavItem.tsx]: why there is an `a` tag inside a `Link`?

**Comment by gregoriusjimmy** [comment]
i think better ganti warna text linknya jadi putih kalau di mobile

File: src/components/modules/Navbar/NavItem.tsx (hunk 1 above)
Comment (falbas) [reply src/components/modules/Navbar/NavItem.tsx]: dari sini
https://github.com/nodefluxio/weber/blob/main/frontend/app/components/modules/Navbar/NavItem.tsx

Comment (gregoriusjimmy) [reply src/components/modules/Navbar/NavItem.tsx]: wkwkw dosa itumah jgn diikutin, gimana klo manfaatin component UnstyledLink?

**Review by gregoriusjimmy — APPROVED** [review]

**Pull request merged**